frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/PullCurrentBranchAction.java

```diff
@@ -0,0 +1,94 @@
+package com.virtuslab.gitmachete.frontend.actions.toolbar;
+
+import static com.virtuslab.gitmachete.frontend.actions.common.ActionUtils.getGitMacheteRepository;
+import static com.virtuslab.gitmachete.frontend.actions.common.ActionUtils.getProject;
+import static com.virtuslab.gitmachete.frontend.actions.common.ActionUtils.getSelectedVcsRepository;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.CommonDataKeys;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.intellij.openapi.project.Project;
+import git4idea.repo.GitRepository;
+import io.vavr.control.Option;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
+
+import com.virtuslab.gitmachete.backend.api.BaseGitMacheteBranch;
+import com.virtuslab.gitmachete.backend.api.SyncToRemoteStatus;
+import com.virtuslab.gitmachete.frontend.actions.common.BasePullBranchAction;
+import com.virtuslab.gitmachete.frontend.datakeys.DataKeys;
+import com.virtuslab.logger.IPrefixedLambdaLogger;
+import com.virtuslab.logger.PrefixedLambdaLoggerFactory;
+
+/**
+ * Expects DataKeys:
+ * <ul>
+ *  <li>{@link DataKeys#KEY_GIT_MACHETE_REPOSITORY}</li>
+ *  <li>{@link DataKeys#KEY_SELECTED_VCS_REPOSITORY}</li>
+ *  <li>{@link CommonDataKeys#PROJECT}</li>
+ * </ul>
+ */
+public class PullCurrentBranchAction extends BasePullBranchAction {
+  private static final IPrefixedLambdaLogger LOG = PrefixedLambdaLoggerFactory.getLogger("frontendActions");
+
+  @Override
+  @UIEffect
+  public void update(AnActionEvent anActionEvent) {
+    super.update(anActionEvent);
+
+    Presentation presentation = anActionEvent.getPresentation();
+    if (!presentation.isEnabledAndVisible()) {
+      return;
+    }
+
+    Option<BaseGitMacheteBranch> currentBranch = getGitMacheteRepository(anActionEvent)
```

Comment:
Use the helper from utils

Reply:
Ain't no ~sunshine~ helper for that yet. You probably meant `getCurrentBranchNameIfManaged` or `getCurrentMacheteNonRootBranch` but none of them is applicable here. :sun_with_face:

There are (most likely will be after I push changes referencing this review) 4 places where we could use `getCurrentMacheteBranch`. WDYT?

Reply:
Ok, let's add a new one then